README.md

```diff
@@ -70,9 +71,9 @@ Terminology
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/numpy-feedstock.svg?style=svg)](https://circleci.com/gh/conda-forge/numpy-feedstock)
-OSX: [![TravisCI](https://travis-ci.org/conda-forge/numpy-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/numpy-feedstock) 
```

Comment:
Ugh, this apparently slipped through due to the current resolution of issue ( https://github.com/conda-forge/conda-smithy/issues/319  ).